Gruntfile.js

```diff
@@ -13,6 +16,55 @@ grunt.initConfig({
 	},
 	jscs: {
 		all: [ "*.js" ]
+	},
+
+	// minifies SVGs
+	svgmin: {
+		options: {
+			plugins: [
+				{
+					removeViewBox: false
+				}, {
+					removeUselessStrokeAndFill: false
+				}
+			]
+		},
+		dist: {
+			files: [ {
+				expand: true,
+				cwd: "svg-source",
+				src: [ "*.svg" ],
+				dest: "svg-min/",
+				ext: ".svg"
+			} ]
+		}
+	},
+
+	// Combines SVGs into single file
+	svgstore: {
+		defaults: {
+			options: {
+				prefix: "icon-", // This will prefix each ID
```

Comment:
We previously decided to follow the jQuery js style guide unless we come up with a strong reason not to http://contribute.jquery.org/style-guide/js/

So comments should be on the line above with a blank line preceding them here and line below